Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -1830,6 +1830,92 @@ def RequestData(self, request, inInfo, outInfo):
         return 1
 
 
+class vortexTIFFCroppedReader(cilBaseCroppedReader, vortexTIFFImageReaderInterface):
+    '''vtkAlgorithm to load and crop a TIFF files
+
+    Example
+    -------
+    This example reads a HDF5 image from the
+    'entry1/tomo_entry/data/data' dataset of the
+    file: data.nxs and crops it to extent (0, 2, 3, 5, 1, 2):
+
+    reader = cilHDF5CroppedReader()
+    reader.SetFileName('data.nxs')
+    reader.SetDatasetName('entry1/tomo_entry/data/data')
+    reader.SetTargetExtent((0, 2, 3, 5, 1, 2))
+    reader.Update()
+    image = reader.GetOutput()
```

Comment:
```suggestion
    This example reads from a list of tiff filenames = tiff_fnames
    and crops it on the z axis to extent (1, 3):

    reader = vortexTIFFCroppedReader()
    reader.SetFileName(tiff_fnames)
    reader.SetTargetZExtent([1,3])
    reader.Update()
    image = reader.GetOutput()
```

Docstring needs updating to something like this.